firebase_admin/messaging.py

```diff
@@ -264,6 +375,42 @@ def send(self, message, dry_run=False):
         else:
             return resp['name']
 
+    def send_all(self, messages, dry_run=False):
+        """Sends the given messages to FCM via the batch API."""
+        if not isinstance(messages, list):
+            raise ValueError('Messages must be an list of messaging.Message instances.')
+        if len(messages) > 100:
+            raise ValueError('send_all messages must not contain more than 100 messages.')
+
+        responses = []
+
+        def batch_callback(_, response, error):
+            exception = None
+            if error:
```

Comment:
This doesn't look right. I'd except `error` here to be the `ApiCallError` raised in `postproc`.